### ISSUE TITLE
Cannot download multiple rasters at the same time

### ISSUE THREAD

**Issue by brmather** [michaelchin/plate-model-manager#1]
To reproduce the error:

```python
pm_manager = PlateModelManager()
muller2019_model = pm_manager.get_model("Muller2019", data_dir="plate-model-repo")
muller2019_model.get_rasters("AgeGrids", [0,1])
```

![Screenshot 2023-11-14 at 12 30 20 pm](https://github.com/michaelchin/plate-model-manager/assets/15915881/3961897d-cb7e-47a8-9a7a-87d06f346bcd)

**Comment by michaelchin** [comment]
This is due to the `jupyter notebook` is interfering with the event loop.
You need to add the following two lines at the beginning of the first code cell to workaround the problem for now.
```
import nest_asyncio
nest_asyncio.apply()
```

Meanwhile, I will see if I can add some code to deal with the jupyter notebook situation.

**Comment by michaelchin** [comment]
fixed in d3e0c56325136abf5d828b12d81b0137bccfa3cd
